src/main/java/org/broadinstitute/consent/http/service/DatabaseDataSetAPI.java

```diff
@@ -172,13 +172,11 @@ public ParseResult overwrite(File dataSetFile, Integer userId) {
         return result;
     }
 
-
+    @Deprecated
     @Override
     public Collection<DataSetDTO> describeDataSets(Integer dacUserId) {
-        Collection<DataSetDTO> dataSetDTOList;
-        if (userIs(UserRoles.RESEARCHER.getRoleName(), dacUserId)) {
-            dataSetDTOList = dsDAO.findDataSetsForResearcher();
-        } else {
+        Collection<DataSetDTO> dataSetDTOList = dsDAO.findDataSets();
+        if (!userIs(UserRoles.RESEARCHER.getRoleName(), dacUserId)) {
```

Comment:
Have not completely removed this old method to ask some follow up questions... In the old behavior, we would check for whether or not the given user is a researcher or admin. (This would short circuit on researcher even if a user was both). If not a researcher, and yes an admin, the dataset list retrieved would be self-selected for certain election properties, upon which properties like `updateAssociationToDataOwnerAllowed`, `isAssociatedToDataOwners`, and `deletable` would be set. These properties are only set here, and are otherwise always null. Should we still be assigning these values based on the same criteria despite wanting to fetch all datasets irregardless of election status?

Reply:
These fields affect whether or not certain buttons on the dataset catalog are disabled

Reply:
Great questions.
In the case of admins/chairs we want to get all datasets regardless of active/inactive. We need this because an admin/chair can enable a currently disabled dataset. These are all admin-level fields so a user with admin permissions on that dataset can execute the available functions
* Admins: Can perform admin functions on all datasets
* Chairs: Can perform admin functions only on datasets that are associated to their DACs. Other datasets should show a null prop so the UI can eventually do the right thing. We might need a story for this since I don't think we've ever made any catalog updates wrt a Chair view.
* Everyone else: Can see only active datasets